### ISSUE TITLE
Split view in the context menu

### ISSUE THREAD

**Issue by snsrkn** [mbnuqw/sidebery#1365]
## Description
As of version 11.6, Floorp has the "split view" feature. The only way to use this feature is to click on "Fixed in split view", in the tabs context menu, but it not visible in Sidebery's tab context menu. 

It would be great to use this feature with Sidebery. 

![image](https://github.com/mbnuqw/sidebery/assets/84017607/e9e2e503-759f-442a-b234-e68bc65bea96)

**Comment by albino1** [comment]
If you shift+right-click you get the normal Firefox menu, does that work with Floorp as well?

**Comment by snsrkn** [comment]
> If you shift+right-click you get the normal Firefox menu, does that work with Floorp as well?

No, it doesn't work. I think shift+right click overrides Floorp and get Firefox's menu.

**Comment by albino1** [comment]
Ah, I see.  That makes sense.
